server/routes/reports/prisoners/remove/removePrisoner.test.ts

```diff
@@ -4,11 +4,11 @@ import { convertReportWithDetailsDates } from '../../../../data/incidentReportin
 import { mockReport } from '../../../../data/testData/incidentReporting'
 import { now } from '../../../../testutils/fakeClock'
 import {
-  AddPrisonerInvolvementRequest,
+  AddStaffInvolvementRequest,
   IncidentReportingApi,
-  PrisonerInvolvement,
+  StaffInvolvement,
   RelatedObjects,
-  UpdatePrisonerInvolvementRequest,
+  UpdateStaffInvolvementRequest,
```

Comment:
these should all still refer to _prisoners_, it’s the new test cases that are about staff

Reply:
also is this IR-819?